src/timePicker/TimePicker.style.tsx

```diff
@@ -1,26 +1,37 @@
 import styled from 'styled-components'
 
-import { color, font, inputBorderSize, space } from '../_utils/branding'
+import { color, font, inputBorderSize, responsiveBreakpoints } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
 
-export const StyledTimePicker = styled.div`
-  & {
+export const StyledTimePickerWrapper = styled.div`
+  width: 100%;
+  ${normalizeHorizontally};
+
+  // On large media, let's fix the size and center the component
+  @media (${responsiveBreakpoints.isMediaLarge}) {
     position: relative;
-    display: inline-flex;
-    justify-content: center;
-    align-items: center;
     width: 320px;
-    max-width: 100%;
-    font-size: 4em;
-    font-weight: 500;
-    line-height: 1;
-    background-color: ${color.white};
-    color: ${color.midnightGreen};
-    border: 1px solid ${color.gray};
-    border-radius: 2em;
-    box-sizing: border-box;
-    cursor: pointer;
-    padding: ${inputBorderSize.focus};
+    left: 50%;
+    transform: translateX(-50%);
   }
+`
+
+export const StyledTimePicker = styled.div`
+  position: relative;
+  width: 100%;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 4em;
```

Comment:
This font-size doesn't follow our scale. Do you think you could come closer by using `brand` or `xxl` sizes?

Reply:
Pixar specs says its xxl (82px). But it doesn't fit on small screens, meaning we should apply the following rule:

`It can’t go on more than 1 line. If data is too long we reduce the font size (see edge case).`

I'll leave this for the realign libraries epic. I'll create a jira